turbo.json

```diff
@@ -1,39 +1,54 @@
 {
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": ["package.json"],
   "pipeline": {
     "build": {
-      "dependsOn": ["^build", "intl-compile", "codegen"],
+      "dependsOn": ["^build", "build:css", "intl-compile", "codegen"],
       "inputs": ["src/**", "package.json"],
-      "outputs": ["dist/**"]
+      "outputs": ["dist/**"],
+      "outputMode": "new-only"
     },
     "build:css": {
-        "outputs": ["src/assets/css/hydrogen.css"]
+      "inputs": ["src/**"],
+      "outputs": ["src/assets/css/hydrogen.css"]
     },
     "codegen": {
       "inputs": ["src/**/*.graphql", "../../packages/**/src/**/*.graphql"],
-      "outputs": ["src/api/generated.ts", "src/index.ts"]
+      "outputs": ["src/api/generated.ts", "src/index.ts"],
+      "outputMode": "new-only"
     },
     "dev": {
       "dependsOn": ["^build", "build:css", "intl-compile", "codegen"],
       "inputs": ["src/**", "package.json"],
-      "outputs": ["dist/**"]
+      "outputs": ["dist/**"],
+      "outputMode": "new-only"
     },
     "watch": {
       "dependsOn": ["build:css", "intl-compile", "codegen"],
-      "cache": false
+      "cache": false,
+      "persistent": true
     },
     "build:intl-cli": {
-      "outputs": ["dist/cli.js"]
+      "inputs": ["src/cli.ts"],
+      "outputs": ["dist/cli.js"],
+      "outputMode": "new-only"
     },
     "intl-extract": {
-      "outputs": ["src/lang/en.json"]
+      "inputs": ["src/**"],
+      "outputs": ["src/lang/en.json"],
+      "outputMode": "new-only"
     },
     "intl-compile": {
+      "inputs": ["src/**"],
       "dependsOn": ["^intl-compile", "intl-extract"],
-      "outputs": ["src/lang/*{Compiled}.json"]
+      "outputs": ["src/lang/*{Compiled}.json"],
+      "outputMode": "new-only"
     },
     "check-intl": {
+      "inputs": ["src/**"],
       "dependsOn": ["build:intl-cli", "intl-compile", "intl-extract"],
-      "cache": false
+      "cache": false,
+      "outputMode": "new-only"
     },
     "storybook": {
       "dependsOn": ["codegen", "intl-compile"],
```

Comment:
This should depend on build:css, too, right?

Reply:
Kind of annoying because it isn't very intuitive but, storybook runs `h2-build` when compiling through webpack so it is not necessary here. We needed to do it this way to get it working with storybooks HMR. I wish we could just put it here and make it explicit but it is hidden away in:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f5b4498fae499d4fe245645baf4ad07a14de1e80/packages/hydrogen-webpack-plugin/index.js#L18-L24

Reply:
I'd really love to write a proper h2 webpack plugin someday.

Reply:
I looked into it with the packages refactor which resulted in this plugin but it wasn't part of the scope so I did not spend much time on it at all.  I'm kinda waning on webpack after using other bundlers. Its so complex and slow 😢